internal/pkg/exec/docker.go

```diff
@@ -52,12 +55,17 @@ const (
 	credStoreECRLogin = "ecr-login" // set on `credStore` attribute in docker configuration file
 )
 
-// Architectures that receive special handling.
+// Operating systems and architectures.
 const (
-	ArmArch   = "arm"
-	Arm64Arch = "arm64"
+	LinuxOS   = "linux"
+	Amd64Arch = "amd64"
+	FmtOSArch = "%s/%s" // Stringified platform.
```

Comment:
It is slightly strange to export a format string :D 

What do you think of instead making this private and creating a function instead?
```go
func DockerBuildPlatform(os, arch string)  string { return fmt.Sprintf("%s/%s", os, arch) }
```

Reply:
Haha, yes, I originally had a "stringifyPlatform" function that I did away with for the const. Will switch back to function.